src/core_plugins/kibana/public/visualize/listing/visualize_listing.html

```diff
@@ -39,7 +39,7 @@
           ng-click="listingController.deleteSelectedItems()"
           tooltip="Delete selected visualizations"
         >
-          <span aria-hidden="true" class="kuiButton__icon kuiIcon fa-trash"></span>
+          <trash-icon-component></trash-icon-component>
```

Comment:
I agree that the component names should not contain "Component". IHMO `<visualize-landing-table>` sounds ok as does `<kui-trash-icon>`.